pkg/preparer/orchestrate_test.go

```diff
@@ -82,7 +84,9 @@ func TestPreparerLaunchesPodsThatHaveDifferentSHAs(t *testing.T) {
 		currentManifest: existing,
 	}
 	newManifest := testManifest(t)
-	success := installAndLaunchPod(newManifest, testPod, logging.DefaultLogger)
+
+	var p *Preparer
```

Comment:
It's never dereferenced anywhere in installAndLaunchPod. Once @petertseng and I add our code, we'll need to actually declare something there.